docs/5_2_scheduled_execution_api.adoc

```diff
@@ -8,379 +8,125 @@ The simulation algorithm signals to the FMU that it supports and has recognized
+* [[clockReference,`clockReference`]] `clockReference`: <<valueReference>> of an <<inputClock>> which shall be activated
 
-[source, C]
-----
-void CallbackIntermediateUpdate(..., fmi3Boolean clocksTicked, ...)
-{
-   fmi3ValueReference outputClockReferences = {7};
-   fmi3Boolean[] clocksActivationState = {fmi3ClockInactive};
-   if (clocksTicked)
-   {
-      // ask FMU if output clock has ticked
-      fmi3GetClocks(... outputClockReferences, &clocksActivationState, ...);
-   }
-   if (clocksActivationState[0])
-   {
-      // schedule task for AperiodicClock
-      Scheduler->ScheduleTask(TaskAperiodic);
-   }
-}
-----
+* [[clockElementIndex,`clockElementIndex`]] `clockElementIndex`: The element index of the <<inputClock>> variable which shall be activated.
+For a scalar <<clock>> variable this must be 0; for array <<clock>> variables, the element clock to activate is specified using the 1-based element index.
+Using the element index 0 means all elements of the <<clock>> variable.
+(Note: If an array has more than one dimension the indices are serialized in the same order as defined for values in <<serialization-of_variables>>).
 
-===== FMU Implementation
+* [[activationTime, `activationTime`]] `activationTime`: value of the <<independent>> variable of the assigned <<inputClock>> tick time latexmath:[\mathbf{t}_i] _[typically: simulation (i.e. virtual) time]_ (which is known to the simulation algorithm).
 
-The FMU implements <<fmi3ActivateModelPartition>> dispatching for every <<inputClock,`input clock`>> so the code might look like this:
++
+The importer schedules calls of <<fmi3ActivateModelPartition>> for each FMU.
+Calls are based on activations of <<inputClock,`input clocks`>>.
+These <<inputClock>> activations can be based on <<clock>> activations from FMU external sources (e.g. <<outputClock,`output clocks`>> of other FMUs).
+The <<inputClock>> activations can also be based on <<outputClock>> ticks of the same FMU.
+The <<fmi3ActivateModelPartition>> function must not be called on <<outputClock,`output clocks`>> of an FMU.
 
-[source, C]
-----
-fmi3Status fmi3ActivateModelPartition(fmi3Instance *instance,
-   fmi3ValueReference clockReference, fmi3Float64 activationTime)
-{
-   switch (clockReference)
-   {
-      case 5:
-         // Input clock 10msClock
-         activateModelPartition10ms(instance, activationTime);
-      case 6:
-         // Input clock AperiodicClock
-         activateModelPartitionAperiodic(instance, activationTime);
-      case 8:
-         // Input clock AperiodicClock
-         activateModelPartition50ms(instance, activationTime);
-      ...
-   }
-}
-----
++
+This is a different timing concept compared to <<fmi3DoStep>> calls for Co-Simulation FMUs or the <<fmi3SetTime>> for Model Exchange FMUs.
```

Comment:
Is this and the following non-normative?

Reply:
It must be explained, that timing in all the interface types have their own definitions.
This is a good point and I think it must be normative to clarify the difference.